book/tutorials/sar/swesarr.ipynb

```diff
@@ -468,7 +492,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!wget --quiet https://n5eil01u.ecs.nsidc.org/SNOWEX/SNEX20_SWESARR_TB.001/2020.02.11/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKKa225H_v01.csv -O {output_dir}/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKuKa225H_v03.csv"
+    "!wget --user=this --password=that --quiet https://n5eil01u.ecs.nsidc.org/SNOWEX/SNEX20_SWESARR_TB.001/2020.02.11/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKKa225H_v01.csv -O {output_dir}/SNEX20_SWESARR_TB_GRMCT2_13801_20007_000_200211_XKuKa225H_v03.csv"
```

Comment:
I'd prefer not to change this. We instructed everyone to set up a ~/.netrc file (), and out CI system that executes and renders the website relies on such a file being there. Adding the user and password flags means this command will fail so the notebook won't render. https://snowex-hackweek.github.io/website/preliminary/earthdata.html#configure-programmatic-access-to-nasa-servers

Reply:
Hi Scott! I absolutely did not mean to change that. I just noticed I accidentally left it in while doing a practice run an hour ago. Give me just a second and I'll make a new pull request